common/api/frontend-tiles.api.md

```diff
@@ -88,18 +88,18 @@ export type GraphicRepresentation = {
 });
 
 // @beta
-export type GraphicRepresentationFormat = "IMDL" | "3DTILES" | string;
+export type GraphicRepresentationFormat = "IMODEL" | "3DTiles" | "CESIUM";
```

Comment:
@danieliborra @arvindsvenkat To clarify, is "3DFT" as an export type still returned by the service? If so I'll add it here